lms/views/admin/__init__.py

```diff
@@ -29,5 +30,13 @@ def flash_validation(request, schema):
     except ValidationError as err:
         request.session.flash(err.messages["form"], "validation")
         return True
-
     return False
+
+
+def error_render_to_response(
+    request, error_message, template, template_args, flash_type="errors"
+):
+    request.session.flash(error_message, flash_type)
+    response = render_to_response(template, template_args, request=request)
```

Comment:
We render to response when rendering a different template that the view's default.